packages/loader/container-loader/src/loader.ts

```diff
@@ -71,7 +71,7 @@ export class RelativeLoader implements ILoader {
                         canReconnect: request.headers?.[LoaderHeader.reconnect],
                         clientDetailsOverride: request.headers?.[LoaderHeader.clientDetails],
                         resolvedUrl: {...resolvedUrl},
-                        version: request.headers?.[LoaderHeader.version],
+                        version: request.headers?.[LoaderHeader.version] ?? undefined,
```

Comment:
This change does not seem to be required

Reply:
request.headers?.[LoaderHeader.version] returns null if not present. So needs to return undefined.